src/domains/dashboard/utils/render-charts-and-menu.ts

```diff
@@ -9,11 +9,13 @@ import { netdataDashboard, options } from "./netdata-dashboard"
 // to reflect our menu system and content
 // TODO: this is a shame - we should fix charts naming (issue #807)
 // ^^ (original comment from gsmox, still valid!)
-function enrichChartData(chart: ChartMetadata) {
-  const parts = chart.type.split("_")
+function enrichChartData(chartName: string, chart: ChartMetadata) {
```

Comment:
I'm not sure about that. Shouldn't it be safer to have a separate function doing this for the Room Overview and leave this here for the NodeView and Agent Dashboard?

Reply:
There is no reason to duplicate code in this case. The type is safely extracted from the name on both node and context charts.

Reply:
Ok lets double check it can be reused for all cases (Overview, NodeView, AgentDashboard)
Something else: why change the function signature to include chartName while chart name is included in ChartMetadata?

Reply:
Also, shouldn't it be something other than ChartMetadata for the RoomOverview case? (If this is true it brings back the question shouldn't it be better a new separate function?)

Reply:
Thanks @VLegakis for getting attention to this. After testing, this change looks ok for me. I was a little unsure, but i've checked on our demo servers, that given chart keys, `key.split(".")[0]` always equals `chart.type`. So we can use that now. It would be good though to double-check in Agent team.

Reply:
> Something else: why change the function signature to include chartName while chart name is included in ChartMetadata?

From my understanding this is a change that allows us to work also when we don't have `chart.type` property, the only place we can get it is by extracting it from `charts[key]`